src/lib/onlyText.ts

```diff
@@ -1,6 +1,5 @@
 import type { ReactNode } from 'react';
 import { Children, isValidElement } from 'react';
-import hasChildren from './hasChildren.js';
```

Comment:
This would be still needed

Reply:
I refactored conditions ⚡ a5728f36863c93e81c8dde83ce9e9ebffbae6df9

I hope it explains you why we don't need `hasChildren()`.

Reply:
Sorry but it's intenional to reuse `hasChildren` as it brings value and it's tested in isolation

Reply:
I see.

I looked that the package uses multiple `isValidElement` calls everywhere. I agree, we should use DRY concept, but another hand we should think about reducing the execution time of the process. We should find balance.

I hope you will optimize code in the future.